src/patternfly/components/TreeView/examples/TreeView.md

```diff
@@ -402,15 +408,16 @@ beta: true
       {{/tree-view-list}}
     {{/tree-view-list-item}}
     {{#> tree-view-list-item tree-view-list-item--IsExpandable="true"}}
-      {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--text="Cost management"}}
+      {{> tree-view-node tree-view-node--type="div" tree-view-node--attribute='tabindex="0"' tree-view-node--HasToggle="true" tree-view-node--type="div" tree-view-node--text="Cost management"}}
     {{/tree-view-list-item}}
     {{#> tree-view-list-item tree-view-list-item--IsExpandable="true"}}
-      {{> tree-view-node tree-view-node--HasToggle="true" tree-view-node--text="Sources"}}
+      {{> tree-view-node tree-view-node--type="div" tree-view-node--attribute='tabindex="0"' tree-view-node--HasToggle="true" tree-view-node--type="div" tree-view-node--text="Sources"}}
```

Comment:
I wonder if we could build all of this into `tree-view-node--HasToggle` or any other attributes that make this need to be a div with a tabindex, etc.

Also you have `tree-view-node--type="div"` twice in this and some others.